components/content/ContentCode.vue

```diff
@@ -4,7 +4,7 @@ const props = defineProps<{
   lang?: string
 }>()
 
-const raw = $computed(() => decodeURIComponent(props.code).replace(/&#39;/g, '\''))
+const raw = computed(() => decodeURIComponent(props.code).replace(/&#39;/g, '\''))
```

Comment:
Thanks a lot for this PR 🙏🏼 

I think we could keep the `$computed` here, did you have a reason for the change?

Reply:
Ah, I see, @danielroe has our back there